### PR TITLE
String Logging

### PR DESCRIPTION
**Pull request by cadenceforney**
makes it so you can write strings to files solves issue #56

### CODE REVIEW

**Review by Vif15 — APPROVED** [review]

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/RobotConfig.kt [hunk 1]
```diff
@@ -113,4 +113,10 @@ object RobotConfig {
         @JvmField
         var BOTSBURGH_FOLDER = File(AppUtil.ROOT_FOLDER, "/BotsBurgh")
     }
+
+    @Config
+    object StringLogging {
+        @JvmField
+        var BOTSBURGH_FOLDER = File(AppUtil.ROOT_FOLDER, "/BotsBurgh")
```
Comment (BD103) [inline TeamCode/src/main/java/org/firstinspires/ftc/teamcode/RobotConfig.kt]: `CsvLogging` and `StringLogging` both have the same `BOTSBURGH_FOLDER` variable. Can we combine them both into a single category? (Maybe named `Logging`?)

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/api/CsvLogging.kt [hunk 2]
```diff
@@ -64,7 +64,7 @@ object CsvLogging : API() {
             val writer = this.fileHash[file]!!
 
             writer.write("${opMode.runtime}")
-            for (i in data) fileHash[file]!!.write(", $i")
+            for (i in data) CsvLogging.fileHash[file]!!.write(", $i")
```
Comment (BD103) [inline TeamCode/src/main/java/org/firstinspires/ftc/teamcode/api/CsvLogging.kt]:
```suggestion
            for (i in data) fileHash[file]!!.write(", $i")
```

Unless I'm missing something, you shouldn't need to qualify this path.

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/api/StringLogging.kt [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+    }
+
+    /**
+     * Writes String data to the targeted file
+     * @param file Name of the file that is being logged to
+     * @param data String data that is being logged
+     */
+    fun writeFile(
+        file: String,
+        data: String,
```
Comment (BD103) [inline TeamCode/src/main/java/org/firstinspires/ftc/teamcode/api/StringLogging.kt]:
```suggestion
        data: Any,
```

You can merge all three variants of `writeFile()` into one if you switch the type of `data` to `Any`, then call `data.toString()` when writing it to a file.

**Review by BD103 — COMMENTED** [review]
Looks pretty good! I have a few nits, but none of them are blocking.

**Review by BD103 — APPROVED** [review]

**Pull request merged**